src/main/scala/is/hail/HailContext.scala

```diff
@@ -174,25 +172,20 @@ object HailContext {
         "org.apache.hadoop.io.compress.GzipCodec"
     )
 
-    sparkContext.uiWebUrl.foreach(ui => info(s"SparkUI: $ui"))
     ProgressBarBuilder.build(sparkContext)
 
-    log.info(s"Spark properties: ${
-      sparkContext.getConf.getAll.map { case (k, v) =>
-        s"$k=$v"
-      }.mkString(", ")
-    }")
-
     val sqlContext = new org.apache.spark.sql.SQLContext(sparkContext)
     val hc = new HailContext(sparkContext, sqlContext, tmpDir, branchingFactor)
+    sparkContext.uiWebUrl.foreach(ui => info(s"SparkUI: $ui"))
     val welcomeMessage =
-      """Welcome to
+      """
+        |Welcome to
         |     __  __     <>__
         |    / /_/ /__  __/ /
         |   / __  / _ `/ / /
         |  /_/ /_/\_,_/_/_/   version """.stripMargin + is.hail.HAIL_PRETTY_VERSION
-    println(welcomeMessage)
-    log.info(welcomeMessage)
+
+    info(welcomeMessage)
```

Comment:
Do we need this here?  Just log the version.  I don't think we need to pretend the user is going to run through Scala anymore.